docs/what_is_the_dev_preview.rst

```diff
@@ -0,0 +1,47 @@
+What is the Raiden Developer Preview
+************************************
+
+*And what is it not*
+
+
+.. toctree::
+  :maxdepth: 2
+
+Disclaimer!
+-----------
+**The Raiden Developer Preview is not at a Minimum Viable Product (MVP) stage, and hence shouldn't be considered safe to use on the Ethereum main net.**
+
+
+Introduction
+------------
+The goal with releasing the Developer Preview is to give developers a chance to get familiar with the
+Raiden API and to figure out what will be possible to do with Raiden, once it's fully ready.
+
+Below is a listing of things that are included in the Developer Preview together with things that are in the roadmap, but are not yet implemented.
+
+
+The Developer Preview includes
+-------------------------------
+
+- Opening, closing and settling of payment channels.
+- Direct and mediated (multi hop) transfers.
+- Automatically joining a token network and connecting to peers
+- A :doc:`REST API <rest_api>` with endpoints for all functionality.
+- Possibility to create token networks for all ERC20 tokens.
+- Restartability of Raiden nodes.
```

Comment:
Here perhaps make a note that restartability means only if case of a proper shutdown